server/utils/cloudFunctions/functions/test/countAdvertisersTests/countAdvertisersFunctions.test.js

```diff
@@ -0,0 +1,400 @@
+                }).catch(err => console.log(err));
+
+            return;
+          });
+        });
+      });
+    });
+    
+    it(SHOULD_INCREMENT +  'for every ad', () => {
+      const advertiser = "adv_c"; 
+      const snapOne = 
+          snapFromJson({advertiser, startDate: "2019-10-15"}, DEV_ADS_PATH);
+      const snapTwo = 
+          snapFromJson({advertiser, startDate: "2019-10-16"}, DEV_ADS_PATH);
+
+      const wrapper = firestoreWrap(devCountAdvertisersOnCreate);
+      
+      return wrapper(snapOne).then(() => {
+        return wrapper(snapTwo).then(() => {
+          DEV_AGGREGATES_COLLECTION
+              .doc("2019")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 2);
+              }).catch(err => console.log(err));
+            
+          return;
+        });
+      }); 
+    });
+
+    it(SHOULD_INCREMENT + 'even if ads have same start date', () => {
+      const advertiser = "adv_d"; 
+      const snapOne = 
+          snapFromJson({advertiser, startDate: "2019-10-15"}, DEV_ADS_PATH);
+      const snapTwo = 
+          snapFromJson({advertiser, startDate: "2019-10-15"}, DEV_ADS_PATH);
+
+      const wrapper = firestoreWrap(devCountAdvertisersOnCreate);
+      
+      return wrapper(snapOne).then(() => {
+        return wrapper(snapTwo).then(() => {
+          DEV_AGGREGATES_COLLECTION
+              .doc("2019")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 2);
+              }).catch(err => console.log(err));
+          
+          return;
+        });
+      }); 
+    });
+
+    it(SHOULD_INCREMENT + 'only for ads in same year', () => {
+      const advertiser = "adv_e"; 
+      const snapOne = 
+          snapFromJson({advertiser, startDate: "2019-10-15"}, DEV_ADS_PATH);
+      const snapTwo = 
+          snapFromJson({advertiser, startDate: "2020-10-15"}, DEV_ADS_PATH);
+
+      const wrapper = firestoreWrap(devCountAdvertisersOnCreate);
+      
+      return wrapper(snapOne).then(() => {
+        return wrapper(snapTwo).then(() => {
+          DEV_AGGREGATES_COLLECTION
+              .doc("2020")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+          
+          DEV_AGGREGATES_COLLECTION
+              .doc("2020")
+              .collection("advertisers")
+              .doc(advertiser).get().then(function(doc) {
+                return assert.equal(doc.data().numberOfAds, 1);
+              }).catch(err => console.log(err));
+
+          return;
+        });
+      }); 
+    });   
+
+    it(SHOULD_INCREMENT + 'only for an advertisers own ads', () => {
```

Comment:
this description is kinda unclear, it's really testing two different advertisers.